test/integration/highlight_editor_spec.mjs

```diff
@@ -1811,6 +1815,9 @@ describe("Highlight Editor", () => {
           }
```

Comment:
Shouldn't we also use `Math.round(y)` in the line above since `y` is divided in the loop? That'd also make it equal to the changes below.